src/globals.ts

```diff
@@ -5,3 +5,13 @@
 
 export const DIALOG_ID = 'password-confirmation-dialog'
 export const MODAL_CLASS = 'modal-mask' // NcModal component root class https://github.com/nextcloud/nextcloud-vue/blob/v7.0.0-beta.2/src/components/NcModal/NcModal.vue
+
+export const LAX = 'lax'
+export const STRICT = 'strict'
+export type PWD_CONFIRMATION_MODE = typeof LAX | typeof STRICT
```

Comment:
What about enum ? :sweat_smile:

Reply:
Cannot be exported to JS I think :(

Reply:
> Cannot be exported to JS I think :(

It can, it is just some kind of object :)

Reply:
Ah ! Will try

Reply:
Done, thanks both :)